libs/designsystem/src/lib/components/modal/footer/modal-footer.component.ts

```diff
@@ -10,7 +10,13 @@ export class ModalFooterComponent {
   @Input()
   snapToKeyboard = false;
 
+  @HostBinding('class.fixed')
+  isFixed: boolean = true;
   @HostBinding('class.inline')
-  @Input()
-  inline = false;
+  isInline: boolean;
+
+  @Input() set type(type: 'inline' | 'fixed') {
+    this.isFixed = type === 'fixed';
+    this.isInline = type === 'inline';
+  }
```

Comment:
Tested it and it seems to work without problems! But I think it might be possible to simplify this a bit. 
```typescript
@HostBinding('class')
@Input() type: 'inline' | 'fixed' = 'fixed';
```

It would also match how we do it in the code otherwise 😄  
Example from `libs/designsystem/src/lib/components/radio/radio.component.ts` ~line 19: 

```typescript
  @HostBinding('class')
  @Input()
  size?: 'xs' | 'sm' | 'md';

```